React/Views/SafeAreaView/RCTSafeAreaView.m

```diff
@@ -41,48 +40,10 @@ - (NSString *)description
 
   return [NSString stringWithFormat:@"%@; safeAreaInsets = %@; appliedSafeAreaInsets = %@>",
                                     superDescription,
-                                    NSStringFromUIEdgeInsets([self safeAreaInsetsIfSupportedAndEnabled]),
+                                    NSStringFromUIEdgeInsets(self.safeAreaInsets),
                                     NSStringFromUIEdgeInsets(_currentSafeAreaInsets)];
 }
 
-- (BOOL)isSupportedByOS
-{
-  return [self respondsToSelector:@selector(safeAreaInsets)];
-}
-
-- (UIEdgeInsets)safeAreaInsetsIfSupportedAndEnabled
-{
-  if (self.isSupportedByOS) {
-    return self.safeAreaInsets;
```

Comment:
We can delete this property because `safeAreaInsetsIfSupportedAndEnabled` always returns `self.safeAreaInsets`.